Elasticsearch/Search/SearchRequest.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CommonBundle\Elasticsearch\Search;
+
+use EMS\CommonBundle\Contracts\Elasticsearch\Search\SearchRequestInterface;
+
+final class SearchRequest implements SearchRequestInterface
+{
+    /** @var array */
+    private $body;
+    /** @var array */
+    private $contentTypes;
+    /** @var int */
+    private $from = 0;
+    /** @var array */
+    private $indexes;
+    /** @var int */
+    private $size = 10;
+    /** @var array */
+    private $sourceIncludes = [];
+    /** @var array */
```

Comment:
nullable

Reply:
nothing is nullable (body, contentTypes, sourcleIncludes, sourceExcludes, indexes) are always arrays. Size and from is always int and version is boolean.